config/samples/_v1_porterconfig.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     getporter.org/testdata: "true"
 spec:
   verbosity: debug
-  defaultSecretsPlugin: kubernetes.secrets
-  defaultStorage: in-cluster-mongodb
+  default-secretsPlugin: kubernetes.secrets
```

Comment:
Seeing this sample here has made me realize that we don't have a schemaVersion/Type for config files. I've created https://github.com/getporter/porter/issues/2542 as a placeholder for us to consider that.